scripts/update-remote-wn-client

```diff
@@ -61,8 +61,15 @@ def download_to_file(uri, outfile):
 
 
 def setup_cas(osg_ca_scripts_dir, cert_dir):
-    "Run osg-ca-manage setupCA"
+    """Run osg-ca-manage setupCA from a git clone of osg-ca-scripts.
 
+    This will write CA certificates into cert_dir.  As a side effect, it will
+    also create a symlink in osg_ca_scripts_dir/etc/grid-security/certificates
+    but we can ignore that.
+
+    """
+    if not cert_dir.endswith("certificates"):  # required by osg-ca-manage
```

Comment:
Just curious but would the check be better as `os.path.basename(cert_dir) == "certificates"`, or is it ok to have a dir names `xyz_certificates` ?